R/defaults.R

```diff
@@ -76,14 +63,16 @@ epidist_stancode.default <- function(data, ...) {
 #' @family defaults
 #' @export
 epidist.default <- function(data, formula = brms::bf(mu ~ 1, sigma ~ 1),
-                            family = brms::lognormal(), fn = brms::brm, ...) {
+                            family = brms::lognormal(), prior = NULL,
+                            fn = brms::brm, ...) {
   epidist_validate(data)
   epidist_family <- epidist_family(data, family)
   epidist_formula <- epidist_formula(
     data = data, family = epidist_family, formula = formula
   )
+  class(family) <- c(class(family), family$family)
   epidist_prior <- epidist_prior(
-    data = data, family = epidist_family, formula = formula
+    data = data, family = family, formula = epidist_formula, prior
   )
```

Comment:
Continues to be ugly that some things need the `formula` and some the `epidist_formula`. Some the `family` and some the `epidist_family`

Reply:
Ah right got you. I wonder if that suggests that the formula should be in the epidist_formula return and so on

Reply:
Yes maybe, I'll comment that on the issue for it. Then `epidist_formula` isn't just a formula etc. though which is sad